.vuepress/public/_redirects

```diff
@@ -203,3 +203,10 @@
 # New "Update and migration" section
 /learn/getting_started/algolia_migration.html       /learn/update_and_migration/algolia_migration.html
 /learn/advanced/updating.html                       /learn/update_and_migration/updating.html
+
+# Move faceting to a new guide
+/learn/advanced/filtering_and_faceted_search.html   /learn/advanced/faceted_search.html
```

Comment:
```suggestion
/learn/advanced/filtering_and_faceted_search.html#faceted-search   /learn/advanced/faceted_search.html
```

Reply:
I think you'd still want to redirect the `/learn/advanced/filtering_and_faceted_search.html` URL